gallery/src/styles/NxTable/NxTableExamples.tsx

```diff
@@ -101,6 +105,19 @@ const NxTablesExamples = () =>
       This example demonstrates what happens when a table is set up to enable scrolling, but does not have enough
       content to cause scrolling.
     </GalleryExampleTile>
+
+    <GalleryExampleTile title="NX Table with Sortable Rows Example"
+                        liveExample={NxTableSortableExample}
+                        codeExamples={NxTableSortableCode}>
+          A demonstration of a <code className="nx-code">nx-table</code> used for columns that can be sorted.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="NX Table with Filter Rows Example"
+                        liveExample={NxTableFilterExample}
+                        codeExamples={NxTableFilterCode}>
+        A demonstration of a <code className="nx-code">nx-table</code> with a header
+        cell that contains a filter. Rows can be filtered depending on the text provided in the input.
```

Comment:
A brief mention that the interactivity is not hooked up in this example would be good.  Same in the sorting one above.

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/2b0c5e0e1a91b46d016604ec83e11a9c5532bb69